scripts/install-chef-suse.sh

```diff
@@ -563,13 +563,6 @@ REPOS_SKIP_CHECKS+=" SUSE-OpenStack-Cloud-6-Pool SUSE-OpenStack-Cloud-6-Updates"
 # FIXME: Update repo doesn't exist yet, can't be checked
 REPOS_SKIP_CHECKS+=" SUSE-Enterprise-Storage-2-Updates"
```

Comment:
I just realize that you can also remove the 5 lines above. I'm pretty sure we also have code that sets REPOS_SKIP_CHECKS elsewhere that can be dropped.